### ISSUE TITLE
 Identify essential data points for analysis.

### ISSUE THREAD

**Issue by st00giez** [444B/ANTwarfare#15]
It looks to me as though the API is the only source for data collection, limiting what the tool can do.

Is there somewhere else we can pull data from? Or use player data? I dont think there is

**Comment by 444B** [comment]
@st00giez That is correct, it does seem like we are limited in the current data tools for now if we just use the warapi

I can think of the following data sources but am not sure how many of them would be valid data sources:
1. Reddit (Comments, likes, memes)
2. Discord ( number of regiments, bot data (if available))
3. Youtube ( number of streams, viewers, likes)
4. Twitch ( number of streams, viewers)

Sadly, most of those are not tied to real time data coming from the game but as a result of it

Maybe we can speak to Siegecamp about getting extra data sources? Not sure they see an incentive
